third_party/README-cppunit.txt

```diff
@@ -0,0 +1,2 @@
+Please note that the cppunit zip file contains the original source distribution,
```

Comment:
tar.gz is not needed. I will remove the src of libevent from third party lib; we can simply store compressed copy of libevent same way as done for cppunit.